### DISCUSSION TITLE
Sea dog - Get a grip!

### DISCUSSION THREAD

**Discussion by MaaiG** [FakeFishGames/Barotrauma#7660] [Ideas and suggestions]
To be honest, AI literally disappoints.
I’m not talking about trying to walk into a wall, not being able to open a door, having trouble putting out a fire or anything.

Today affecting me so much how wordly-wise submariners are psyched in the half-flooded rooms, which leads to their death.

https://user-images.githubusercontent.com/93348499/145069019-ccd4975e-ad3b-4a51-999b-9b4fe10a69ed.mp4

It may be interesting to add mechanics to the game, where the more experienced characters will be more stress-resistant and professional.
Assuming AI can do that.

**Comment by mLuby** [comment]
This example is contrived to fail. If that button is closer to the hatch (or better yet just use the HatchWithButtons), the AI will automatically try to use the button to open the hatch. And of course they're trying to escape the flooding room, wouldn't you? Also not even human players could get through that hatch without a ladder or full flooding.

What does this have to do with the [Seawolf class](https://en.wikipedia.org/wiki/Seawolf-class_submarine)? Title is confusing.

**Comment by MaaiG** [comment]
The hatch, the button, the ladder don’t matter (They’re is the remains of the description of the some bug). The behavior of the characters is important! Instead of staying in a air bubble, they dive to death!

> What does this have to do with the Seawolf class? Title is confusing.

Nothing at all.
Ok - sea dog! ))
